products/bash/release-ace-integration-server.sh

```diff
@@ -13,33 +13,46 @@
 #   -n : <namespace> (string), Defaults to "cp4i"
 #   -r : <is_release_name> (string), Defaults to "ace-is"
 #   -i : <is_image_name> (string), Defaults to "image-registry.openshift-image-registry.svc:5000/cp4i/ace-11.0.0.9-r2:new-1"
+#   -z : <tracing_namespace> (string), Defaults to "cp4i"
 #
 # USAGE:
 #   With defaults values
-#     ./release-ace-is.sh
+#     ./release-ace-integration-server.sh
 #
 #   Overriding the namespace and release-name
-#     ./release-ace-is -n cp4i -r cp4i-bernie-ace
+#     ./release-ace-integration-server -n cp4i -r cp4i-bernie-ace
 
-function usage {
-    echo "Usage: $0 -n <namespace> -r <is_release_name> -i <is_image_name>"
-    exit 1
+function usage() {
```

Comment:
Any reason for the addition of the '()' here?

Reply:
fixed